socketshark/utils.py

```diff
@@ -5,14 +5,16 @@
 
 import aiohttp
 
-
 from . import constants as c
 
 
 def _get_rate_limit_wait(log, resp, opts):
     """
-    Returns the number of seconds we should wait given a 429 HTTP response and
-    HTTP options.
+    Return the number of seconds we should wait if rate limited.
+
+    Args:
```

Comment:
`log` for completeness?

Reply:
FWIW, this was addressed in a rebase at https://github.com/closeio/socketshark/pull/83.